ionic generate failing "standalone is not defined"

I have a brand-new Ionic Angular NgModule App scaffolded using the Ionic CLI.

I updated **@ionic/angular-toolkit** to version 11.0.0 and can no longer generate anything using the CLI.

**Issue:**

using ``ionic generate page/component`` results in ``standalone is not defined``

**Process:**

Use ``ionic generate component components/temp`` 

Result:

``
standalone is not defined
[ERROR] Could not generate component.
``

Is this a bug or should I have changed something? Since this update is a week old, I am having a hard time finding anything that can help.

Thanks in advance for any help.

--- a/packages/schematics/component/schema.d.ts
+++ b/packages/schematics/component/schema.d.ts
@@ -12,4 +12,5 @@ export interface Schema {
   export?: boolean;
   entryComponent?: boolean;
   type?: string;
+  standalone?: boolean;
 }

--- a/packages/schematics/component/schema.json
+++ b/packages/schematics/component/schema.json
@@ -85,6 +85,11 @@
       "type": "string",
       "description": "Adds a developer-defined type to the filename, in the format \"name.type.ts\".",
       "default": "Component"
+    },
+    "standalone": {
+      "type": "boolean",
+      "description": "Specifies if the component should be standalone",
+      "default": false
     }
   },
   "required": []

